inspirehep/base/static/less/base/header.less

```diff
@@ -629,3 +596,173 @@ a.external-link::after {
     content: " \f08e";
     font-family: FontAwesome;
 }
+
+// Mixin to set background color
+.collection-background-color(@background-color;) {
+    background-color: @background-color;
+}
+
+@media (max-width: 991px) {
+    #top-search-bar-full {
+        display: none;
+    }
+}
+
+@media (min-width: 991px) {
+    #top-search-bar-small {
+        display: none;
+    }
+}
+
+// To prevent shrinking search field after applying typeahead 
+.navbar-form .input-group .form-control {
+  width: 100%;
+}
+
+.navbar-form .input-group .input-group-btn {
+  width: 1%;
+}
+
+.navbar-default .navbar-collapse,
+.navbar-default .navbar-form {
+    border: 0;
+}
+
+#top-search-bar-small {
+
+    float: none;
+    margin: 0 auto;
+
+    #searchform .navbar-form {
+        margin-top: 11px;
+        margin-bottom: 11px;
+    }
+}
+
+#top-search-bar-full {
+    @media (min-width: 1200px) {
+        margin-left: 3%;
+    }
+}
+
+
+#top-search-bar-full, #top-search-bar-small {
+
+    padding: 0;
+
+    #searchform .navbar-form {
+        box-shadow: none;
+    }
+
+    #searchform-input-group {
+        width: initial;
+        font-size: larger;
+    }
+
+    #search-form-input {
+        min-width: 75px;
+        border: 0;
+        border-radius: 0;
+    }
+
+    button {
+        padding-left: 20px;
+        padding-right: 20px;
+        border: 0;
+        font-size: 16px;
+
+        // Top bar search button colours
+        &.literature {
+            .collection-background-color(@literature-collection);
+        }
+
+        &.authors {
+            .collection-background-color(@authors-collection);
+        }
+
+        &.conferences {
+            .collection-background-color(@conferences-collection);
+        }
+
+        &.jobs {
+            .collection-background-color(@jobs-collection);
+        }
+
+        &.experiments {
+            .collection-background-color(@experiments-collection);
+        }
+
+        &.institutions {
+            .collection-background-color(@institutions-collection);
+        }
+
+        &.journals {
+            .collection-background-color(@journals-collection);
+        }
+
+        &.data {
+            .collection-background-color(@data-collection);
+        }
+    }
+}
+
+// TO-DO: fix these..
+@media (min-width: 767px) {
```

Comment:
would be good to use the standard break points that we use overall (with their variable name)